### ISSUE TITLE
Add extent of DoMethod to IIR

### ISSUE THREAD

**Issue by dominichofer** [MeteoSwiss-APN/dawn#641]
To write a back-end the extents of DoMethods are needed.
These are currently not part of the IIR.
So to save every back-end from the hassle of computing it, I propose we add the extents to the IIR.

**Comment by Stagno** [comment]
I agree that we should export them to serialized IIR. We export the accesses of statements, so why not those of a DoMethod?
